tests/archery_damage_test.cpp

```diff
@@ -0,0 +1,78 @@
+/*
+ * Researching what seem to be generally agreed on values
+ * for bow power required for guaranteed game kills, these are some breakpoints.
+ * While many source use KE values, there is a broad consensus that momentum is related to damage.
+ * FPS  ft-lbs  slug-ft-s  Game
+ * 136  19.01  0.28  Turkeys
+ * 153  24  0.315  Whitetails at close range
+ * 195  39  0.401  Whitetails
+ * 249  63.73  0.512  Elk
+ * 278  79.44  0.572  Moose
+ *
+ * The concept is to bracket these threshods with various bows using standard hunting loadouts.
+ */
+
+#include <string>
+
+#include "catch/catch.hpp"
+#include "ballistics.h"
+#include "creature.h"
+#include "game_constants.h"
+#include "item.h"
+#include "monster.h"
+#include "projectile.h"
+
+static void test_projectile_attack( std::string target_type, bool killable,
+                                    dealt_projectile_attack &attack, std::string weapon_type )
+{
+    for( int i = 0; i < 10; ++i ) {
+        monster target{ mtype_id( target_type ), { 0, 0, 0 } };
```

Comment:
clang-tidy doesn't like this
```suggestion
        monster target{ mtype_id( target_type ), tripoint_zero };
```